composer.json

```diff
@@ -28,7 +28,7 @@
   },
```

Comment:
Missing: update to the version constraint for the `composer/composer` package in  `require-dev`.

Reply:
What would you like this to read? It's currently `*`, which means "any version".

Reply:
Well, if we're dropping support for Composer < 2.0, it should become `^2.0` I'd say... (or `^2.2` if we'd go for a Composer 2.2 minimum).